smelli/classes.py

```diff
@@ -121,6 +123,12 @@ def __init__(self, eft='SMEFT', basis=None,
           observables and each key is a string that serves as user-defined
           name. For each item of the dictionary, a custom likelihood will be
           computed.
+        - add_measurements: a dictionary in which each key is a likelihood
+          and each value is a list of measurements to be added to that
+          likelihood.
+        - remove_measurements: a dictionary in which each key is a likelihood
```

Comment:
Since you're not using type hints, can you be more specific about the types of these inputs? Are the keys strings ("a likelihood")?

Reply:
I decided not to include type hints because they were not used in the rest of the code. I expanded the description, I hope it's clearer now.